Form.css

```diff
@@ -0,0 +1,24 @@
+.Form {
```

Comment:
Under most conventions, CSS selectors are `kebab-case`. In my opinion, we should assure consistency by following a style guide.